lib/ide/ast/impl/src/lib.rs

```diff
@@ -16,6 +18,10 @@ use serde::Serialize;
 use shapely::*;
 use uuid::Uuid;
 
+
+#[derive(Clone, Debug, Default, Serialize, Deserialize)]
+pub struct IDMap(pub Vec<(Span, ID)>);
```

Comment:
Name does not follow rust convention. Should be `IdMap`

Reply:
Plus no spaces after `,`. Please see our code style guide - this PR has **a lot** of code not following it :( I'm not putting comments everywhere as it makes just more sense to ask you to read it and apply globally.